### PR TITLE
[Flutter Parent] Add students and sorting in course chooser bottom sheet

### PR DESCRIPTION
**Pull request by StephenBrough**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022110&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022116&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/flutter_parent/lib/l10n/app_localizations.dart [hunk 1]
```diff
@@ -328,6 +328,9 @@ class AppLocalizations {
   String get attachmentFailed => Intl.message('Failed. Tap for options.',
       desc: 'Short message shown on a message attachment when uploading has failed');
 
+  String forWhom(String studentShortName) =>
+      Intl.message('for $studentShortName', desc: 'Describes who is in a course, i.e. for Bill');
```
Comment (CalvinKern) [inline apps/flutter_parent/lib/l10n/app_localizations.dart]: Needs 'args' and 'name'

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/flutter_parent/lib/l10n/app_localizations.dart [hunk 2]
```diff
@@ -328,8 +328,12 @@ class AppLocalizations {
   String get attachmentFailed => Intl.message('Failed. Tap for options.',
       desc: 'Short message shown on a message attachment when uploading has failed');
 
-  String forWhom(String studentShortName) =>
-      Intl.message('for $studentShortName', desc: 'Describes who is in a course, i.e. for Bill');
+  String courseForWhom(String studentShortName) => Intl.message(
+        'for $studentShortName',
+        desc: 'Describes for whom a course is for (i.e. for Bill)',
+        args: [studentShortName],
+        name: 'forWhom',
```
Comment (CalvinKern) [inline apps/flutter_parent/lib/l10n/app_localizations.dart]: The name has to match the method name exactly. 'courseForWhom'

Comment (StephenBrough) [reply apps/flutter_parent/lib/l10n/app_localizations.dart]: Lol, changed the name after setting that up. ARGH.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022148&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022149&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022150&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1022151&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**